boilerplate/app/models/root-store/root-store.ts

```diff
@@ -1,12 +1,9 @@
 import { Instance, SnapshotOut, types } from "mobx-state-tree"
-import { NavigationStoreModel } from "../../navigation/navigation-store"
 
 /**
  * A RootStore model.
  */
-export const RootStoreModel = types.model("RootStore").props({
-  navigationStore: types.optional(NavigationStoreModel, {}),
-})
+export const RootStoreModel = types.model("RootStore").props({})
```

Comment:
I think this might cause issues with the model generator. It looks for `.props({` and then inserts below (See https://github.com/infinitered/ignite-bowser/blob/master/src/commands/generate/model.ts#L49) so I think the closing `})` should stay on it's own line. You might have to add a `// prettier-ignore` to avoid the auto formatting.